internal/bminventory/inventory.go

```diff
@@ -447,6 +447,10 @@ func (b *bareMetalInventory) RegisterCluster(ctx context.Context, params install
 		params.NewClusterParams.VipDhcpAllocation = swag.Bool(true)
 	}
 
+	if params.NewClusterParams.Operators.Ocs.Enabled == nil {
```

Comment:
You need to add some tests on Create/Update cluster.
I assume that `params.NewClusterParams.Operators` can be nil